optimizely/optimizely.py

```diff
@@ -383,6 +388,19 @@ def is_feature_enabled(self, feature_key, user_id, attributes=None):
                                     decision.variation,
                                     user_id,
                                     attributes)
+        self.notification_center.send_notifications(enums.NotificationTypes.FEATURE_EXPERIMENT,
+                                                  feature_key, user_id, attributes,
+                                                    decision.experiment, decision.variation)
+      else:
+        audience = None
+        if decision.experiment is not None and \
+                decision.experiment.audienceIds is not None and \
+                len(decision.experiment.audienceIds) > 0:
+          audience_id = decision.experiment.audienceIds[0]
```

Comment:
@thomaszurkan-optimizely Hi, I have a question here. In the case where the rollout rule/experiment has multiple audience IDs, and the user didn't qualify for the first audience, but for some other audience id in the list. Do we still send the audience at index 0 in send_notifications method?